Sources/Proxy/Core/PiecewiseFunctionProxy/index.js

```diff
@@ -3,6 +3,34 @@ import macro from 'vtk.js/Sources/macro';
 import vtkPiecewiseFunction from 'vtk.js/Sources/Common/DataModel/PiecewiseFunction';
 import vtkPiecewiseGaussianWidget from 'vtk.js/Sources/Interaction/Widgets/PiecewiseGaussianWidget';
 
+import Constants from './Constants';
+
+const { Mode, Defaults } = Constants;
+
+// ----------------------------------------------------------------------------
+
+function applyPointsToPiecewiseFunction(points, range, pwf) {
+  const width = range[1] - range[0];
+  const rescaled = points.map(([x, y]) => [x * width + range[0], y]);
+
+  pwf.removeAllPoints();
+  rescaled.forEach(([x, y]) => pwf.addPoint(x, y));
+  pwf.sortAndUpdateRange();
+}
+
+// ----------------------------------------------------------------------------
+
+function applyNodesToPiecewiseFunction(nodes, range, pwf) {
+  const width = range[1] - range[0];
+  const rescaled = nodes.map((n) =>
+    Object.assign({}, n, { x: n.x * width + range[0] })
+  );
+
+  pwf.removeAllPoints();
+  pwf.set({ nodes: rescaled }, true);
```

Comment:
There is not setNodes? Are we sure we don't need to do additional actions like modified(), build()... ?

Reply:
I think you can call set points with sharpness and so on...

Reply:
There's `setNodeValue`, but no `setNodes`. I modeled this after `applyGaussianToPiecewiseFunction`, which uses `set()` directly. `sortAndUpdateRange` is the PiecewiseFunction equivalent of build+modified as far as I can tell.

Reply:
Should we add `setNodes`, which will implicitly call `sortAndUpdateRange`, to PiecewiseFunction?

Reply:
ok, I guess this is fine for now. 
Still we should have a single method inside the piecewiseFunction to not externalize the responsibility like I did in the VolumeRepProxy with the default...

Reply:
Yes I would think so...

Reply:
I'll make that a separate PR then.

Reply:
ok